web/assets/js/views/PhysicalEvaluation-0.3-blood-donor.js

```diff
@@ -262,8 +262,11 @@ PMI.views['PhysicalEvaluation-0.3-blood-donor'] = Backbone.View.extend({
             block.find('.modification-select').show();
         }
         if (modification === 'refusal' || modification === 'pandemic' || modification === 'colostomy-bag') {
-            block.find('.modification-affected input, .modification-affected select, .modification-manual input:checkbox').each(function() {
-                $(this).attr('disabled', true);
+            block.find('.modification-affected input:text, .modification-affected select').each(function () {
+                $(this).valChange('').attr('disabled', true);
```

Comment:
ℹ️ Didn't recognize this method, so I looked it up. Neat.

https://github.com/all-of-us/healthpro/blob/989f17cbc8f3c6fbe2660b14ca46576945e92c7e/web/assets/js/app.js#L259-L269